### ISSUE TITLE
Comments on the peer review for the HW2

### ISSUE THREAD

**Issue by glederrey** [lfaucon/ada-homeworks#5]
Hi guys,

I'm just posting some comments for the review of the HW2.. It may help you to become even better.. =)
1. Your scraping is good.. But as Michele said during the lecture on Friday,  it's good to put the results into a file such that you don't call ISA again and again.. =)
2. You're missing the statistical test for the comparison of the specialization. It would have given more information than the average of months.. =)
3. For the graph with the Specialization, be careful with the colors.. If a colorblind person is looking at your graph, he won't be able to distinguish the colors.. It's not a big problem here since you have good labels.. But it's just for you and your future papers.. 
4. For the bonus, you're missing the graph that was asked.. =P

In conclusion, you did a very good job. Your comments are really clear, your code is clean. It was very easy to read. At the end, you just missed the stats test for the Specialization.. 

Good job guys!

**Comment by lfaucon** [comment]
Thanks for the review!
